Flaky test exposes time formatting bug

Saw this when I just happened to refresh at the right time of the day:

```
  125 passing (78ms)
  1 failing

  1) IITC.utils.unixTimeToString
       should return the time in HH:mm:ss format when the timestamp is for today:

      AssertionError: expected '00:00:20' to equal '24:00:20'
      + expected - actual

      -00:00:20
      +24:00:20
      
      at Context.<anonymous> (file:///home/runner/work/ingress-intel-total-conversion/ingress-intel-total-conversion/test/utils.spec.js:194:27)
      at process.processImmediate (node:internal/timers:476:21)
```

--- a/test/utils.spec.js
+++ b/test/utils.spec.js
@@ -183,7 +183,7 @@ describe('IITC.utils.unixTimeToString', () => {
     const timestamp = now.getTime();
     const timeString = IITC.utils.unixTimeToString(timestamp);
 
-    const timeFormatter = new Intl.DateTimeFormat('en-US', {
+    const timeFormatter = new Intl.DateTimeFormat('sv-SE', {
       hour: '2-digit',
       minute: '2-digit',
       second: '2-digit',

